interfaces/builtin/opencl.go

```diff
@@ -0,0 +1,53 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2016 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package builtin
+
+import (
+	"github.com/ubuntu-core/snappy/interfaces"
+)
+
+const openclConnectedPlugAppArmor = `
+# Description: Can access opencl. 
+# Usage: reserved
+
+  # OpenCL Installable Client Driver (ICD) Loader
+  /etc/OpenCL/vendors/** r,
```

Comment:
@niemeyer - this is a weird area. I'm assuming this interface is only for classic since this isn't part of the os snap and as you mentioned, this directory may be empty or not exist on the system at all since it requires additional debs to be installed on the classic system. Doing a dynamic check to see if the opencl binaries are installed and if /etc/OpenCL/vendors/* could be a way to do this, but then I worry about the precedent that would set. We might end up with hundreds of little interfaces to optional debs which seems very uncontrolled and not designed.... Why aren't these files and whatever binaries are needed not part of the shipping snap? (sorry if this is obvious-- I'm not familiar with opencl).